CONTRIBUTING.md

```diff
@@ -0,0 +1,180 @@
+# Contributing
+
+Thank you for your interest in contributing to the SparkPost Developer Site!
+
+## API reference
```

Comment:
These headings _might_ work a little better as actions? Something like
* Updating the API Reference
* Editing the Table of Contents
etc?

Reply:
Hm I misunderstood the grouping here so my suggestion won't work as-is, but I think "verbing" those headings would help if you can figure out a way to do something like that... :)